O/ORTools/build_tarballs.jl

```diff
@@ -161,7 +164,15 @@ products = [
 ]
 
 # Dependencies that must be installed before this package can be built
-dependencies = Dependency[]
+dependencies = [
+    # OR-Tools 9.12 starts depending on CMake 3.28.
+    # OR-Tools 9.12 starts depending on CMake 3.28.
+    BuildDependency(PackageSpec(; name="CMake_jll", version = v"3.28.1"))
```

Comment:
https://docs.binarybuilder.org/stable/build_tips/#Dependencies-for-the-target-system-vs-host-system

Reply:
And to be extra explicit: https://github.com/JuliaPackaging/Yggdrasil/blob/ddf863dea81a134cba44f5005fce32d985c7ada3/S/SuiteSparse/common.jl#L94